### PR TITLE
Python examples

### PR DESCRIPTION
**Pull request by garnaat**
This supersedes #21.  I have merged the branch started by @dstuffd and am extending it to include more examples.

- [x] application-load-balancer
- [x] chat-app
- [x] classic-load-balancer
- [x] custom-resource
- [ ] ecs
  - [x] cluster (ported but blocked by #23)
  - [x] ecs-loadbalanced-service
  - [x] ecs-service-with-advanced-alb-config
  - [x] ecs-service-with-task-networking
  - [x] ecs-service-with-task-placement (ported but issue with ecs.BuiltInAttributes)
  - [x] fargate-load-balanced-service
  - [x] fargate-service-with-auto-scaling
- [ ] elasticbeanstalk
  - [ ] elasticbeanstalk-bg-pipeline
  - [ ] elasticbeanstalk-environment
- [x] lambda-cron
- [ ] my-widget-service
- [ ] resource-overrides
- [ ] static-site
- [x] stepfunctions-job-poller

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by Doug-AWS** [comment]
Don't forget to update the parent README.md to list your new examples in a table in a Python section.

File: python/stepfunctions/app.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+        definition = submit_job\
+            .next(wait_x)\
+            .next(get_status)\
+            .next(is_complete
+                  .when(sfn.Condition.string_equals(
+                      "$.status", "FAILED"), job_failed)
+                  .when(sfn.Condition.string_equals(
+                      "$.status", "SUCCEEDED"), final_status)
+                  .otherwise(wait_x))
```
Comment (garnaat) [inline python/stepfunctions/app.py]: This formatting is a bit weird but its the best I've been able to come up with so far that is readable and doesn't drive my linter crazy.

**Comment by rix0rrr** [comment]
I think you are done

**Comment by garnaat** [comment]
I would like to merge these now and open separate issues for any additional examples we may want to create or translate.

**Comment by Doug-AWS** [comment]
I'll update the parent README.md to list these new examples in a table in a Python section.

**Pull request merged**